mailgreen/controller/subscription_controller.py

```
@@ -0,0 +1,52 @@
+
+router = APIRouter(prefix="/subscriptions", tags=["subscriptions"])
+
+
+class SenderCount(BaseModel):
+    sender: str
+    name: str
+    count: int
+
+
+@router.post("/sync", response_model=List[SubscriptionOut])
+async def sync_subscriptions(user_id: UUID, db: Session = Depends(get_db)):
```

Ruff (0.11.9)

Redefinition of unused `sync_subscriptions` from line 10

(F811)